src/views/NoticeOfWithdrawal.vue

```diff
@@ -0,0 +1,1042 @@
+import { Certify } from '@/components/common'
+import PlanOfArrangement from '@/components/NoticeOfWithdrawal/PlanOfArrangement.vue'
+import RecordToBeWithdrawn from '@/components/NoticeOfWithdrawal/RecordToBeWithdrawn.vue'
+import ReferenceNumber from '@/components/NoticeOfWithdrawal/ReferenceNumber.vue'
+import StaffPayment from '@/components/NoticeOfWithdrawal/StaffPayment.vue'
+import { ConfirmDialog, StaffRoleErrorDialog, PaymentErrorDialog, ResumeErrorDialog, SaveErrorDialog }
+  from '@/components/dialogs'
+import { CommonMixin, DateMixin, FilingMixin, ResourceLookupMixin } from '@/mixins'
+import { LegalServices } from '@/services/'
+import { FilingStatus, SaveErrorReasons } from '@/enums'
+import { FilingCodes, FilingTypes, StaffPaymentOptions } from '@bcrs-shared-components/enums'
+import { ConfirmDialogType, StaffPaymentIF } from '@/interfaces'
+import { DocumentDelivery } from '@bcrs-shared-components/document-delivery'
+import { useBusinessStore, useConfigurationStore, useRootStore } from '@/stores'
+
+  @Component({
+    components: {
+      Certify,
+      ConfirmDialog,
+      PlanOfArrangement,
+      DocumentDelivery,
+      StaffRoleErrorDialog,
+      PaymentErrorDialog,
+      ResumeErrorDialog,
+      RecordToBeWithdrawn,
+      ReferenceNumber,
+      StaffPayment,
+      SaveErrorDialog,
+      SbcFeeSummary
+    }
+  })
+export default class NoticeOfWithdrawal extends Mixins(CommonMixin, DateMixin, FilingMixin, ResourceLookupMixin) {
+    // Refs
+    $refs!: {
+      confirm: ConfirmDialogType,
+      certifyRef: Certify
+    }
+
+    @Getter(useConfigurationStore) getAuthWebUrl!: string
+    @Getter(useRootStore) getBusinessEmail!: string
+    @Getter(useBusinessStore) getLegalName!: string
+    @Getter(useConfigurationStore) getPayApiUrl!: string
+    @Getter(useRootStore) getUserInfo!: any
+    @Getter(useRootStore) isRoleStaff!: boolean
+
+    // enum for template
+    readonly FilingCodes = FilingCodes
+
+    // variables for Certify component
+    certifiedBy = ''
+    isCertified = false
+    certifyFormValid = false
+
+    // variables for POA component
+    hasPlanOfArrangement = false
+    hasComeIntoEffect = false
+
+    // variable for Reference Number component
+    referenceNumberValid = true
+    referenceNumber = ''
+
+    // variables for Document Delivery component
+    documentDeliveryValid = true
+    documentOptionalEmail = ''
+
+    // variables for staff payment
+    staffPaymentValid = false
+    staffPaymentData = { option: StaffPaymentOptions.NONE } as StaffPaymentIF
+
+    // variables for displaying dialogs
+    staffRoleErrorDialog = false
+    resumeErrorDialog = false
+    saveErrorReason = null as SaveErrorReasons
+    paymentErrorDialog = false
+
+    // other variables
+    totalFee = 0
+    dataLoaded = false
+    loadingMessage = ''
+    filingId = 0 // id of this Notice if Withdrawal filing
+    filingToBeWithdrawn: any // id of filing to be withdrawn
```

Comment:
Should be a string. just to be safe here

Reply:
I prefer "string" instead of "any".

However, since it's an id, I'm thinking it's actually a number. But it might be easier to use in code as a string.

Reply:
Got it, updated